src/vaadin-combo-box-dropdown-wrapper.html

```diff
@@ -170,33 +173,57 @@
       }
 
       _openedChanged(opened, items, loading) {
+        if (!this.$.dropdown) {
+          if (!opened) {
+            return;
+          } else {
+            this._initDropdown();
+          }
+        }
         // Do not attach if no items
         // Do not dettach if opened but user types an invalid search
         this.$.dropdown.opened = !!(opened && (loading || this.$.dropdown.opened || items && items.length));
       }
 
-      ready() {
-        super.ready();
-        // IE11: when scrolling with mouse, the focus goes to the scroller.
-        // This causes the overlay closing due to defocusing the input field.
-        // Prevent focusing the scroller by setting `unselectable="on"`.
-        if (/Trident/.test(navigator.userAgent)) {
-          this._scroller.setAttribute('unselectable', 'on');
-        }
+      _initDropdown() {
+        const template = this.shadowRoot.querySelector('#dropdown-template');
+        template.if = true;
+        template.render();
+
+        this.$.dropdown = this.shadowRoot.querySelector('#dropdown');
+        this.$.dropdown.$.overlay = this.$.dropdown.shadowRoot.querySelector('#overlay');
```

Comment:
Nit: there should not be a need in this one, as it is defined in the scope of the `dropdown` reference set before.

Reply:
So it seems. Removed.